searx/utils.py

```diff
@@ -321,7 +320,7 @@ def int_or_zero(num):
     return convert_str_to_int(num)
 
 
-def is_valid_lang(lang):
+def is_valid_lang(lang) -> Optional[Tuple]:
```

Comment:
When specifying `List` or `Tuple` type hints, I think we should always specify the element types e.g. `Tuple[bool, str, str]]`. For this method Pyright infers `tuple[Literal[True], str, str] | None` revealing that the boolean is always True ... depending on what the method is used for we could remove the boolean ... I dunno.

Reply:
> t the boolean is always True

Yes.. I would to avoid too much code changes in this PR.
To remove this always `True` value, it requires to change some engines:
* https://github.com/searxng/searxng/blob/f240a67bd7df88f38ebadf93dcccaa0f557bddaf/searx/search/processors/online_dictionary.py#L31-L38
* https://github.com/searxng/searxng/blob/f240a67bd7df88f38ebadf93dcccaa0f557bddaf/searx/engines/dictzone.py#L30

Reply:
That's understandable.